src/coreclr/vm/corelib.h

```diff
@@ -169,7 +169,6 @@ DEFINE_CLASS(ATTRIBUTE,             System,                 Attribute)
 
 
 DEFINE_CLASS(BINDER,                Reflection,             Binder)
```

Comment:
```suggestion
```
Can this be deleted too?

Reply:
The `Binder` class is still part of some signatures for other functions that are used - for example:
https://github.com/dotnet/runtime/blob/a53ec50f18af59d864d582d561165ac21ea25089/src/coreclr/vm/metasig.h#L470
https://github.com/dotnet/runtime/blob/a53ec50f18af59d864d582d561165ac21ea25089/src/coreclr/vm/corelib.h#L529